scala/scala.bzl

```diff
@@ -192,7 +205,7 @@ SourceJars: {srcjars}
         jvm_flags=",".join(["-J" + flag for flag in ctx.attr.jvm_flags]),
         resource_src=",".join([f.path for f in ctx.files.resources]),
         resource_dest=",".join(
-          [_adjust_resources_path(f.path)[1] for f in ctx.files.resources]
+          [_adjust_resources_path_by_general_rules(f.path)[1] for f in ctx.files.resources]
```

Comment:
the `general_rules` keeps the original `_adjust_resources_path` behavior. Since this uses internal implementation to strip resources path I didn't want to break behavior